samples/sample_multi_transcode/src/sample_multi_transcode.cpp

```diff
@@ -829,31 +844,31 @@ mfxStatus Launcher::CreateSafetyBuffers()
 
 void Launcher::Close()
 {
-    while(m_pSessionArray.size())
+    while(m_pThreadContextArray.size())
```

Comment:
I don't like all this code in a Close(). I think that all that should be handled something like:
```
std::vector<std::unique_ptr<ThreadTranscodeContext>> m_pThreadContextArray;
std::vector<std::unique_ptr<GeneralAllocator>>       m_pAllocArray;

Close()
{
  m_pThreadContextArray.clear();
  m_pAllocArray.clear();
  ...
}
```
But that's definitely out of scope of this commit and this PR. If you guys agree I suggest @vilichev to implement the change after this PR will be merged.

Reply:
this code releases memory in reverse order... not sure if there is any sense in this